apps/marsha/templates/services/nginx/ingress.yml.j2

```diff
@@ -14,6 +14,8 @@ metadata:
 {% if prefix in acme_enabled_route_prefix %}
     cert-manager.io/issuer: "{{ acme_issuer_name }}"
 {% endif %}
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "{{ marsha_nginx_websocket_proxy_read_timeout }}"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "{{ marsha_nginx_websocket_proxy_send_timeout }}"
```

Comment:
This configuration will impact all connections/locations received by nginx, No side effect is expected?

Reply:
By the ingress. And after in the nginx deploy with marsha we can reconfigure a lower timeout if needed. The first closing the connection will close the connection for all other proxies.